zappa/handler.py

```diff
@@ -583,7 +583,17 @@ def handler(self, event, context):
                         )
 
                     if response.data:
-                        if (
+                        if settings.BINARY_SUPPORT and response.headers.get(
+                            "Content-Encoding"
+                        ):
+                            # We could have a text response that's gzip
+                            # encoded. Therefore, we base-64 encode it.
+                            # Related: https://github.com/zappa/Zappa/issues/908
+                            zappa_returndict["body"] = base64.b64encode(
+                                response.data
+                            ).decode("utf-8")
```

Comment:
Nit: `"utf-8"` is redundant here in Py3 and can be safely removed

Reply:
Removed the switched `.decode("utf-8")` -> `.decode()` in a rebase. Thanks!